### PR TITLE
docs: Review and update blogpost Hydra

### PR DESCRIPTION
**Pull request by tacurran**
## Checklist

- [x ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
- [x] I have read the [security policy](../security/policy)
- [x ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x ] I have added necessary documentation within the code base (if appropriate)

### CODE REVIEW

**Review by aeneasr — CHANGES_REQUESTED** [review]
Please remove the directories `.idea` and revert the changes to `package.json` and `package-lock.json`. You can remove things while keeping the local files using `git rm --cached .idea`. You should also add `.idea` to your [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c).

File: src/pages/blog/run-oauth2-server-open-source-api-security.md [hunk 1]
```diff
@@ -117,10 +115,19 @@ several guiding principles:
...
+```shell
+$ sudo netstat -atuln | grep '9000\|9001\|9010\|9020'
+```
+Note 'netstat' on the MAC does not support all options used in Lunix and Windows. The 'lsof' command ($ man -k lsof) augments some of netstat missing functionality.
 
+For Microsoft Windows 10, use the following command:
 ```shell
-$ sudo netstat -tuplen | grep '9000\|9001\|9010\|9020'
-# On windows, use the following command:
 # netstat -an | findstr /r "9000 9001 9010 9020"
```
Comment (aeneasr) [inline src/pages/blog/run-oauth2-server-open-source-api-security.md:131]: Hashtag generally denote comments. I think the windows shell uses `>` as a shell indicator. So this should probably be:

```suggestion
> netstat -an | findstr /r "9000 9001 9010 9020"
```

File: src/pages/blog/run-oauth2-server-open-source-api-security.md [hunk 2]
```diff
@@ -131,25 +138,35 @@ container.
 
 ```shell
 $ docker ps | grep 'hydra'
-# On windows, use the following command:
+$ docker kill hydra
+$ docker kill --signal=HUP hydra
+```
+For Microsoft Windows use
+```shell
 # docker ps | findstr "hydra"
```
Comment (aeneasr) [inline src/pages/blog/run-oauth2-server-open-source-api-security.md:146]: See comment above

File: src/pages/blog/run-oauth2-server-open-source-api-security.md [hunk 3]
```diff
@@ -117,10 +115,19 @@ several guiding principles:
...
+For Linux
+```shell
+$ sudo ss -atuln | grep '9000\|9001\|9010\|9020'
+```
+For Apple MacOS (BASH)
+ 
+```shell
+$ sudo netstat -atuln | grep '9000\|9001\|9010\|9020'
+```
+Note 'netstat' on the MAC does not support all options used in Lunix and Windows. The 'lsof' command ($ man -k lsof) augments some of netstat missing functionality.
```
Comment (aeneasr) [inline src/pages/blog/run-oauth2-server-open-source-api-security.md:127]: I assume these should be paragraphs? If so, please separate them with newlines:

````suggestion

For Linux

```shell
$ sudo ss -atuln | grep '9000\|9001\|9010\|9020'
```

For Apple MacOS (`/bin/bash` and `/bin/zsh`)
 
```shell
$ sudo netstat -atuln | grep '9000\|9001\|9010\|9020'
```

Note 'netstat' on the MAC does not support all options used in Lunix and Windows. The 'lsof' command ($ man -k lsof) augments some of netstat missing functionality.

````

File: src/pages/blog/run-oauth2-server-open-source-api-security.md [hunk 4]
```diff
@@ -131,25 +138,35 @@ container.
...
+Docker containers so the containers can talk to one another.
 
 ```shell
 $ docker network create hydraguide
 ```
+The result will be something like:
+```shell
+641a26284ff2f8ee4580988371b91923d6711e20aa964ebbdf5b2e4b4f2592b8
+```
+The next section explains how to set up the PostgreSQL database system.
```
Comment (aeneasr) [inline src/pages/blog/run-oauth2-server-open-source-api-security.md:161]: Newline code blocks for better readability:

````suggestion
The result will be something like:

```shell
641a26284ff2f8ee4580988371b91923d6711e20aa964ebbdf5b2e4b4f2592b8
```

The next section explains how to set up the PostgreSQL database system.
````

File: src/pages/blog/run-oauth2-server-open-source-api-security.md [hunk 5]
```diff
@@ -179,63 +195,64 @@ always need to use that secret to access the database.
...
-database will be used to persist and query data. **ORY Hydra prevents data
-leaks** as only token signatures are stored in the database. For a valid token,
-both payload and signature are required.
+The database URL must point to the Postgres container that was created above. The database will be used to persist and query data. **ORY Hydra prevents data leaks** as only token signatures are stored in the database. For a valid token, both payload and signature are required.
 
 ```shell
 $ export DSN=postgres://hydra:secret@ory-hydra-example--postgres:5432/hydra?sslmode=disable
 ```
+The result will be something like:
+```shell
```
Comment (aeneasr) [inline src/pages/blog/run-oauth2-server-open-source-api-security.md:217]:
````suggestion

```shell
````

File: src/pages/blog/run-oauth2-server-open-source-api-security.md [hunk 6]
```diff
@@ -248,14 +265,16 @@ $ docker run -d \
...
 ```
 
 ### Is it alive?
 
 This is easy to answer, just check the docker logs! Or,
 [open the health check](http://127.0.0.1:9001/health/ready) , which should show
-you ok.
+the following in the browser page:
+
+{"status":"ok"}
```
Comment (aeneasr) [inline src/pages/blog/run-oauth2-server-open-source-api-security.md:277]: Wrap in codeblock:

````suggestion
```
{"status":"ok"}
```
````

File: src/pages/blog/run-oauth2-server-open-source-api-security.md [hunk 7]
```diff
@@ -267,15 +286,37 @@ time="2017-06-29T21:26:34Z" level=info msg="Setting up http server on :4444"
...
 
 ```shell
 $ docker run --rm -it \
-  oryd/hydra:v1.4.2 \
+  oryd/hydra:v1.8.5 \
   help
 ```
+This command produces an overview of the CLI as follows:
+```shell
+Run and manage ORY Hydra
```
Comment (aeneasr) [inline src/pages/blog/run-oauth2-server-open-source-api-security.md:299]:
````suggestion
This command produces an overview of the CLI as follows:

```shell
Run and manage ORY Hydra
```

**Review by aeneasr — CHANGES_REQUESTED** [review]

**Comment by tacurran** [comment]
@aeneasr , I think if you merge this npm will update the package-lock automatically

File: src/pages/blog/run-oauth2-server-open-source-api-security.md (hunk 2 above)
Comment (aeneasr) [reply src/pages/blog/run-oauth2-server-open-source-api-security.md:151]:
```suggestion
> docker ps | findstr "hydra"
```

**Pull request merged**